app/templates/components/ui-table/cell/cell-sponsor-image.hbs

```diff
@@ -1 +1 @@
-<img class="ui tiny image" src="{{if this.record this.record '/images/placeholders/Other.jpg'}}" alt="Event logo">
+<img width="40" height="40" class="ui image" src="{{if this.record this.record '/images/placeholders/Other.jpg'}}" alt="Event logo">
```

Comment:
shouldn't it better to do ```max-width:40px; max-height: auto```.so that the proportion of image will not be distorted

Reply:
@daretobedifferent18 i think it should be max-height: 40px and max-width:auto

Reply:
@maze-runnar yes :+1:

Reply:
updated it.